config/locales/server.en.yml

```diff
@@ -5858,3 +5858,380 @@ en:
       duplicate_ids: "has duplicate ids"
       reserved_id: "has a reserved keyword as id: %{id}"
       unsafe_description: "has an unsafe HTML description"
+
+  languages:
```

Comment:
It might be worth including only languages supported by Discourse to reduce the translation cost.